README.md

```diff
@@ -8,19 +8,21 @@
 
 TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. Full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
 
-- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
-- **Sync & Execute**: easily sync & run local data & code in the cloud
+- **Easy to use**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
```

Comment:
good question.

- What are target audiences for the docs?
- What precedence do said audiences take?

For release week, I was thinking mostly driven by views :eyes: rather than bookmarks :star:

1. newbies: no cloud account (10k+ :eyes:)
2. mlops & devops: have cloud accounts and hate setup time (1k+ :eyes:)
3. ml engineers & data scientists: a mix of (1) & (2), don't need separate category

Reply:
@DavidGOrtega what would be your recommendation for the 1st one?

Reply:
The first issue that I find is logically, when I read it It sounds to me that we are offering the `machine` and not the `task`. 
Aside that the real problem to me is that "easy" is nice but why should I change my current aws thingy to use this. 
To me the principal motivation to do so is to know that this `task` allows me to do heavy ops in spot instances hassle-free so I would focus on what the task.
I would not put it in the first place and more after `No cloud vendor lock-in` since I would have introduced the clouds  and I would slightly rephrase it to:

**Easy to use**: Launch a task without infrastructure knowledge or having to be a Devops expert, anyone can do it.

Reply:
current order:

- Easy to use - newbie friendly
- Lower cost - expert friendly
- No cloud vendor lock-in - expert friendly
- Seamless developer experience - neutral
- No waste - expert friendly

is this your proposed order @DavidGOrtega?

- Lower cost - expert friendly
- Easy to use - newbie friendly
- No cloud vendor lock-in - expert friendly
- Seamless developer experience - neutral
- No waste - expert friendly

Reply:
> _expert friendly_

What somebody calls “FinOps” I guess... 😅

Reply:
reworked this with a feature-USP split; please re-review :pray: